modules/javafx.graphics/src/main/java/javafx/scene/layout/Border.java

```diff
@@ -389,6 +389,20 @@ public Border(@NamedArg("strokes") BorderStroke[] strokes, @NamedArg("images") B
         hash = result;
     }
 
+    /**
+     * A convenience factory method for creating a solid border with a single {@code Paint}. The border will use the
```

Comment:
Suggestion to rephrase as:
`A convenience factory method for creating a solid Border by specifying only the {@code Paint} for BorderStroke.`